tests/ARCTokenization.Tests/TestUtils.fs

```diff
@@ -55,12 +57,17 @@ module CvParam =
             (Param.getParamValue cvpActual)
         )
 
-    let structuralEquality (cvpExpectec : CvParam) (cvpActual : CvParam) =
-        termNamesEqual cvpExpectec cvpActual
-        accessionsEqual cvpExpectec cvpActual
-        refUrisEqual cvpExpectec cvpActual
-        valuesEqual cvpExpectec cvpActual
-        
+    let structuralEquality (cvpExpected : CvParam) (cvpActual : CvParam) =
+        termNamesEqual cvpExpected cvpActual
+        accessionsEqual cvpExpected cvpActual
+        refUrisEqual cvpExpected cvpActual
+        valuesEqual cvpExpected cvpActual
+
+    let termEquality onto (cvpExpected : CvParam) (cvpActual : CvParam) =
+        let cvtExp = CvTerm.create(cvpExpected.Accession, cvpExpected.Name, cvpExpected.RefUri)
+        let cvtAct = CvTerm.create(cvpActual.Accession, cvpActual.Name, cvpActual.RefUri)
+        let synos
```

Comment:
seems like something was left out unintentionally here

Reply:
Yeah, it's still WIP. Changing it to draft therefore...